test/unit/transforms/util/githubApiRequests.test.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import GitHubAPI from "../../../../src/config/github-api";
+import { compareCommitsBetweenBaseAndHeadBranches } from "../../../../src/transforms/util/githubApiRequests";
+import { getLogger } from "../../../../src/config/logger";
+
```

Comment:
nit: the indents in this and the json files are different to the others (2 spaces vs 4 space tab)

Reply:
Again, auto formatting on save.